lib/Voice/gateway.ex

```diff
@@ -136,6 +141,7 @@ defmodule Alchemy.Voice.Gateway do
   def websocket_terminate(why, _conn_state, state) do
     Logger.debug("Voice Gateway for #{state.guild_id} terminated, "
                  <> "reason: #{inspect why}")
+
```

Comment:
Is this extra line of whitespace from automatic formatting?

If not, let's just remove it, and run elixirfmt afterwards anyways.

Reply:
That was accidentally added since I fixed this issue for my own fork of Alchemy first, so I had to clean up two lines I added there. I'll fix it as well.